manifests/community_modules.pp

```diff
@@ -7,15 +7,22 @@
   Optional[String]      $revision = undef,
 ) {
   case $type {
-    'hg':    { $_packages = ['mercurial'] }
-    'git':   { $_packages = ['git'] }
-    default: { $_packages = [] }
+    'hg':      { $_packages = ['mercurial'] }
+    'git':     { $_packages = ['git'] }
+    'portage': { $_packages = ['net-im/prosody-modules'] }
+    default:   { $_packages = [] }
   }
   ensure_packages($_packages)
-  -> vcsrepo { $path:
-    ensure   => $ensure,
-    provider => $type,
-    source   => $source,
-    revision => $revision,
+  case $type {
+    'portage': { }
```

Comment:
```
manifests/community_modules.pp:17:manifest_whitespace_closing_brace_before:ERROR:there should be a single space or newline before a closing brace
```

The CI fails (line:line:check:reason) because you should use one of:

```suggestion
    'portage': {}
```

or:

```suggestion
    'portage': {
    }
```

Yes, the error message is not helpful :disappointed: